docker-tests/tests/docker_tests/test_docker_compose_quick_start.py

```diff
@@ -115,6 +115,15 @@ def test_trigger_dag_and_wait_for_result(default_docker_image, tmp_path_factory,
         # Before we proceed, let's make sure our DAG has been parsed
         compose.execute(service="airflow-dag-processor", command=["airflow", "dags", "reserialize"])
 
+        # Verify API server health endpoint is accessible and returns valid response
+        health_response = api_request(
+            "GET",
+            path="monitor/health"
+        )
```

Comment:
It _should_ fix the `ruff` formatting error in CI:
```suggestion
        health_response = api_request("GET", path="monitor/health")
```